diffkemp/simpll/Utils.cpp

```diff
@@ -152,6 +153,12 @@ std::string dropSuffix(std::string Name) {
     return Name.substr(0, Name.find_last_of('.'));
 }
 
+/// Remove everything behind the last dot ('.'). Assumes that hasSuffix returned
+/// true for the name.
+StringRef dropSuffix(StringRef Name) {
+    return Name.substr(0, Name.find_last_of('.'));
+}
```

Comment:
Just wondering if we could reuse the implementation of `std::string dropSuffix(std::string Name)` (it's not nice to have two identical implementations of two functions).

Reply:
Will need to reconstruct one `StringRef` in `Utils::namesMatch` to access `StringRef::startswith_lower` but doing so should be fine since we need to create a copy of the parameters anyway.